LocalSettings.php

```diff
@@ -359,14 +359,16 @@
 $wgGroupPermissions['chatmod']['viewpmlog'] = true;
 $wgGroupPermissions['autopatrol']['autopatrol'] = true;
 $wgGroupPermissions['interwiki-admin']['interwiki'] = true;
+$wgGroupPermissions['template-editor']['template-editor'] = true;
```

Comment:
template editor isn't included in core by default, its a protection level. This has to be defined in $wgRestrictionLevels.

Reply:
https://github.com/Test-Wiki/mediawiki/blob/b68e06ad316c50bbcd96c893e74700188fca94ac/LocalSettings.php#L371